### ISSUE TITLE
SVG within foreignObject needs namespace

### ISSUE THREAD

**Issue by xiaoshidetaikuai** [exupero/saveSvgAsPng#200]
foreignObject中包含svg,svg中包含image时保存图片，image中的图片不能被保存

**Comment by exupero** [comment]
Can you provide a running example using JSFiddle or something similar?

**Comment by xiaoshidetaikuai** [comment]
like this，
```
<svg width='300' height='300'>
    <foreignObject width='50' height='50'>
         <svg width ='50' height='50'>
             <image xlink:href='path'  width='50' height='50'><image>
         </svg>
     </foreignObject>
<svg>
```
but I tried foreignObject`s svg add namespace can export successfully

**Comment by exupero** [comment]
Thanks. I just published version 1.4.11 with a fix.
